lvm

```diff
@@ -0,0 +1,155 @@
+#!/bin/sh
+
+# Expose various types of information about lvm2
+#
+# Usage: lvm <options>
+#
+# Options:
+#
+# -g for used and free space of logical volume groups
+# -p for used and free space of physical volumes.
+# -s for the percentage usage of the snapshots
+# -t for the percentage usage of the thin pools
+#
+# * * * * *   root lvm -g | sponge /var/lib/prometheus/node-exporter/lvm.prom
+#
+# This will expose every minute information about the logical volume groups
+#
+# Author: Badreddin Aboubakr <badreddin.aboubakr@cloud.ionos.com>
+
+
+set -eu
+
+display_usage() {
+  echo "This script must be run with super-user privileges."  
+  echo "Usage: ./lvm options"
+  echo "Options:"
+  echo "Expose various types of information about lvm2"
+  echo "Use -g for used and free space of logical volume groups."
+  echo "use -p for used and free space of physical volumes."
+  echo "Use -s for the percentage usage of the snapshots."
+  echo "Use -t for the percentage usage of the thin pools."
+} 
+
+if [ "$(id -u)" != "0" ]; then
+  display_usage
```

Comment:
I would suggest echoing the error message about needing to run as root, minus the rest of the usage help. Since this is technically an error (you want it to be treated as such by cron), echo the message to stderr.